ui/src/core/index.ts

```diff
@@ -126,19 +126,39 @@ export function generateCore() {
 			*/
 
 			const accessor = parseAccessor(key);
+			let lastElementIndex = accessor.length - 1
 			let stateRef = userState.value;
-			for (let i = 0; i < accessor.length - 1; i++) {
+			
+			// Check if the accessor is meant for deletion.
+			const isDeletion = accessor[lastElementIndex].charAt(0) === '-';
+			const isAddition = accessor[lastElementIndex].charAt(0) === '+';
+			
+			if (isDeletion || isAddition) {
+				// Remove the prefix for processing.
+				accessor[lastElementIndex] = accessor[lastElementIndex].substring(1);
```

Comment:
Also, I'm seeing the prefix is in the last element. Do you think it's possible to have it at the start? That way we can first split the key in "op" and "accessor".

For example,

`+myobj.abc.def`

op: `+`
accessor: `["myobj", "abc", "def"]`

Reply:
It's doable per se, but I think that there're going to be issues with reliably and efficiently "carrying" this identifier from children mutations' `get_mutations_as_dict()` upwards to the first call. I'll see what I can do. Will be keeping you updated on this.

Reply:
Great point, please look into this and let's weigh in the pros and cons of both and make a decision.

Reply:
Managed to make it work this way, as per [latest commit](https://github.com/streamsync-cloud/streamsync/pull/214/commits/e2f9fbde939762400c71c3de70d4d9389530f9fb).